BuildAddOn.py

```diff
@@ -271,46 +276,61 @@ def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, pla
 
     elif platformName == 'MAC':
         subprocess.call ([
-            'cp', '-r',
+            'cp', '-R',
             sourceFolder / f'{addOnName}.bundle',
             packageFolder / f'{fileName}.bundle'
         ])
 
 
+def GetDevKitBuildNum (devKitData, version, platformName):
+    url = devKitData[platformName][version]
+    buildNum = url.split('/')[-2]
+    return buildNum
+
+
 # Zip packages
-def PackageAddOns (args, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
+def PackageAddOns (args, devKitData, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
     Check7ZInstallation ()
 
     for version in acVersionList:
-        if args.release:
+        if args.allLocalizedVersions:
             for languageCode in languageList:
                 CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo', languageCode, True)
         else:
             CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'Debug')
             CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, platformName, 'RelWithDebInfo')
         
-        buildType = 'Release' if args.release else 'Daily'
-        subprocess.call ([
-            '7z', 'a',
-            str (packageRootFolder.parent / f'{addOnName}-{version}_{buildType}_{platformName}.zip'),
-            str (packageRootFolder / version / '*')
-        ])
+        if args.allLocalizedVersions:
+            buildType = 'Release'
+            subprocess.call ([
+                '7z', 'a',
+                str (packageRootFolder.parent / f'{addOnName}-{version}_{buildType}_{platformName}.zip'),
+                str (packageRootFolder / version / '*')
+            ])
+        else:
+            buildType = 'Daily'
+            buildNum = GetDevKitBuildNum (devKitData, version, platformName)

```

Comment:
what should be the fileName if we used a local APIDevKit? This _GetDevKitBuildNum_ works only with the config file URL-s, which are released DevKits, right? (Also, we could add this info in the function name, like: _GetDevKitBuildNumFromURL_)

Reply:
Added argument for specifying DevKit build number when local DevKit is used